feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/send/amount/SelectSendViewModel.kt

```diff
@@ -152,13 +154,19 @@ class SelectSendViewModel(
 
     private val sendInProgressFlow = MutableStateFlow(false)
 
-    private val originAssetFlow = originAsset.flatMapLatest(interactor::assetFlow)
+    private val originAssetFlow = originChainAsset.flatMapLatest(interactor::assetFlow)
         .shareInBackground()
 
-    private val commissionAssetFlow = originChain.flatMapLatest(interactor::commissionAssetFlow)
-        .shareInBackground()
+    val originFeeMixin = feeLoaderMixinFactory.createGenericChangeableFee<OriginGenericFee>(
+        originAssetFlow,
+        coroutineScope,
+        configuration = Configuration(
+            initialState = Configuration.InitialState(
+                supportCustomFee = true
+            )
```

Comment:
I think better approach is to accept supportCustomFee: Flow<Boolean> instead of having a initial state and a setter. Its just easier to use

Reply:
It seems to me that we can leave it this way for now, because during the next scopes we will support custom fee everywhere, and this logic will be removed